base/timing.jl

```diff
@@ -104,6 +104,19 @@ function gc_page_utilization_data()
     return Base.unsafe_wrap(Array, page_utilization_raw, JL_GC_N_MAX_POOLS, own=false)
 end
 
+# must be kept in sync with `src/gc-stock.h``
+const FULL_GC_REASONS = [:FULL_GC_REASON_SWEEP_ALWAYS_FULL, :FULL_GC_REASON_FORCED_FULL_SWEEP,
```

Comment:
these should get some documentation on what they mean.

Reply:
See [comment](https://github.com/JuliaLang/julia/pull/55826#discussion_r1777514633).